src/Property.hpp

```diff
@@ -17,3 +19,5 @@ class Property
     int getPurchasePrice() const;
     std::string getName() const;
 };
+
+using Properties = std::vector<Property>;
```

Comment:
In my opinion we shouldn't hold copy of Property, better if we hold vector of pointers to Property. `using PropertyPtr = std::shared_ptr<Property>;`. After this used, `Property(const Property &)` can stay `delete` :-).